Jenkinsfile2

```diff
@@ -0,0 +1,93 @@
+#!groovy
+//Only one build can run
+properties([disableConcurrentBuilds()])
+
+def getRegions() {
+  return [
+    'us-east-1', 'us-east-2', 'us-west-1', 'us-west-2', 'ap-east-1', 'ap-south-1',
+    'ap-southeast-1', 'ap-southeast-2', 'ap-northeast-1', 'ap-northeast-2', 'ap-northeast-3',
+    'ca-central-1', 'cn-north-1', 'cn-northwest-1', 'eu-central-1', 'eu-west-1', 'eu-west-2',
+    'eu-west-3', 'eu-north-1', 'me-south-1', 'sa-east-1'
+  ]
+}
+
+pipeline {
+  agent {
+    label 'master'
+  }
+  options {
+    buildDiscarder(logRotator(numToKeepStr: '5', artifactNumToKeepStr: '5'))
+    timestamps()
+  }
+  parameters {
+    booleanParam(   name: 'UpdateALB', defaultValue: false, description: 'ALB TG weights will be updated unly if checkbox selected')
+    choice(         name: 'BlueWeight',  choices: [0, 1, 2, 3, 4, 5, 6, 7, 8, 9], description: 'Set the weight of ALB TG')
+    choice(         name: 'GreenWeight', choices: [0, 1, 2, 3, 4, 5, 6, 7, 8, 9], description: 'Set the weight of ALB TG')
```

Comment:
nitpick, does it make sense to set default values?
also, should `GreenWeight` be dependent on `BlueWeight`? like, `GreenWeight` + `BlueWeight` == 10?

Reply:
There is no default value in choice parameter, but the  first item in the list is always default.

Reply:
so one of *Weigth parameters should be in DESC ordering, right?

Reply:
Yes, you're right